### PR TITLE
Create Game Class and Author Class

### PR DESCRIPTION
**Pull request by Fene-87**
### In this project milestone, I have accomplished the following:
- Create a Game class in a separate .rb file.
- Define all Game class properties visible in the diagram and set them up in the constructor method.
- Implement "can_be_archived?" method in the Game class.
- Create an Author class with an association to the Item class in a separate .rb file.
- Define all Author class properties visible in the diagram and set them up in the constructor method.

### CODE REVIEW

File: classes/game.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    @name = name
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    @items = []
+  end
+
+  private
+
+  def can_be_archived?
+    super && last_played_at > 2
```
Comment (shasherazi) [inline classes/game.rb:17]: the variable `last_played_at` is in date format. To check whether it is older than 2 years or not you will have to subtract the `publish_date` from `Date.now` and check if the gap is greater than two years. You can see it how I did it in the `Movie` class.

**Review by shasherazi — CHANGES_REQUESTED** [review]
Great effort, however, there is still room for improvement. See comments.

File: classes/game.rb (hunk 1 above)
Comment (Fene-87) [reply classes/game.rb:17]: It is not present in your movie class.

Comment (shasherazi) [reply classes/game.rb]: Oh I am really sorry @Fene-87 . I meant to say `Item` class. See the lines I am quoting. 3652 is the number of days in 10 years. You can do the same to calculate 2 years.
https://github.com/Fene-87/catalog_of_things/blob/dev/classes/item.rb#L34-L36

**Review by shasherazi — APPROVED** [review]
Great work; everything looks good to me. Time to merge.

**Pull request merged**